src/core/stdcpp/array.d

```diff
@@ -0,0 +1,91 @@
+﻿/**
+ * D header file for interaction with C++ std::array.
+ *
+ * Copyright: Manu Evans 2014 - 2018.
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Authors:   Manu Evans
+ * Source:    $(DRUNTIMESRC core/stdcpp/array.d)
+ */
+
+module core.stdcpp.array;
+
+alias array = std.array;
```

Comment:
That's not necessary

Reply:
It really is!

Reply:
Can you elaborate ?

Reply:
https://forum.dlang.org/thread/lppkkcsyvlvhebmlapat@forum.dlang.org
Knock yourself out!

Reply:
TL;DR, Walter says.

Reply:
TL;DR, slightly longer: you can't import 2 such modules because the namespace scopes conflict with eachother. Also, the namespace called 'std' conflicts with phobos. Basically, we just need to alias all these things out into the module scope (where they should be anyway) so that anything is accessible.